jscl.lisp

```diff
@@ -207,8 +207,7 @@
 
 
 (defun bootstrap (&optional verbose)
-  (let ((*features* (list* :jscl :jscl-xc *features*))
```

Comment:
If I look at `*features*` inside the runtime at JSCL,  I see this:

```
(:MOP :TYPES :JSCL :COMMON-LISP)
```

What do you mean by not appending host features in jscl? I think that was just during cross compilation right?

Reply:
I see, seems that my change is not fully correct and I'll fix it.

I don't think host features should be available even during cross compilation. For example, `alexandria` use `#+sbcl ...`, if `SBCL` is available at cross compilation time, it will generate code that JSCL doesn't support and fails to load in JSCL.

Reply:
So this could be then 
`(let ((*features* (list* :jscl :jscl-xc))`
?

Reply:
I think so, and I had some misunderstanding about where I'm supposed to bind `*features*` (you spotted it in `with-compilation-environment`. I'll fix those!